geoalchemy2/types/__init__.py

```diff
@@ -196,6 +197,20 @@ def check_ctor_args(geometry_type, srid, dimension, use_typmod, nullable):
         return geometry_type, srid
 
 
+@compiles(_GISType, "mariadb")
+def get_col_spec(self, *args, **kwargs):
+    if self.geometry_type is not None:
+        spec = "%s" % self.geometry_type
+    else:
+        spec = "GEOMETRY"
+
+    if not self.nullable or self.spatial_index:
+        spec += " NOT NULL"
+    if self.srid > 0:
+        spec += " SRID %d" % self.srid
+    return spec
+
+
 @compiles(_GISType, "mysql")
 def get_col_spec(self, *args, **kwargs):
```

Comment:
```suggestion
@compiles(_GISType, "mysql")
@compiles(_GISType, "mariadb")
def get_col_spec(self, *args, **kwargs):
```
I think these decorators can also be combined, can't they?

Reply:
Of course!  I should have thought of that. Let me test it and see if it works!

Reply:
Thanks! That's very nice like this :)